apps/cli/src/tui/app.tsx

```
@@ -514,6 +514,15 @@ export function App({
           paddingRight={width >= TRANSCRIPT_PADDING_MIN_WIDTH ? 1 : 0}
         >
           <TranscriptList transcript={state.transcript} />
+          {/* Settled toolActivity groups, painted live inside the scrollbox so mid-turn
+          scrollback includes them. pendingTool (below) stays pinned outside. After
+          flushToolActivity, toolActivity is [] and this region unmounts in the same
+          update that pushLine's the muted transcript entries — no double paint. */}
+          {renderLiveToolActivity(state.toolActivity).map((line, index) => (
+            <text key={index} fg={theme.muted}>
```

Biome (2.5.7)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)